inc/template.php

```diff
@@ -1639,6 +1639,11 @@ function tpl_actiondropdown($empty = '', $button = '&gt;') {
         'user_tools' => array('login', 'register', 'profile', 'admin'),
     );
 
+    $evt = new Doku_Event('TEMPLATE_MOBILETOOLS_DISPLAY', $action_structure);
+    if ($evt->advise_before()) {
+        $action_structure = $evt->data;
```

Comment:
$action_structure is passed by reference anyway.  There should be no need to reassign here.